### PR TITLE
Added --first-batch and --last-batch options to `cubi-tk snappy pull-sheets` (#154)

### LINKED ISSUE #154: cubi-tk snappy pull-sheets needs a --min-version
**Is your feature request related to a problem? Please describe.**
At the moment, we have to remove/comment out lines when we want to process later batches only.

**Describe the solution you'd like**
Add a command as exists for pulling/pushing data.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by Nicolai-vKuegelgen**
Added --first-batch and --last-batch options to `cubi-tk snappy pull-sheets`. 
All samples / lines outside of specified batches will be commented out automatically.

This should fix #154

### CODE REVIEW

File: cubi_tk/snappy/pull_sheets.py [hunk 1]
```diff
@@ -234,7 +247,12 @@ def on_visit_process(self, process, node_path, study=None, assay=None):
             )
 
 
-def build_sheet(config: PullSheetsConfig, project_uuid: typing.Union[str, UUID]) -> str:
+def build_sheet(
+    config: PullSheetsConfig,
+    project_uuid: typing.Union[str, UUID],
+    min_batch: int,
```
Comment (holtgrewe) [inline cubi_tk/snappy/pull_sheets.py:253]: Make consistent with arg names first batch / last batch.

**Comment by holtgrewe** [comment]
This does not handle the case of parents being sequenced after the index, does it?

File: cubi_tk/snappy/pull_sheets.py (hunk 1 above)
Comment (Nicolai-vKuegelgen) [reply cubi_tk/snappy/pull_sheets.py]: done

**Comment by Nicolai-vKuegelgen** [comment]
> This does not handle the case of parents being sequenced after the index, does it?

Probably not - I haven't had cases like that yet so I'm not how they should be handled. In cases of new data the batch number of old entries is usually updated to the new batch and doing the same in a case like this would probably solve this issue as well? But I don't know if that would be done or not.

**Comment by holtgrewe** [comment]
This has logic to ensure that families are complete, I believe.

https://github.com/bihealth/cubi-tk/blob/371a8d3260a537cdada7b0a70b124cda52c031d8/cubi_tk/snappy/parse_sample_sheet.py

**Pull request merged**